### PR TITLE
[GJ-13] Refine docs on Velox backend

### PR DESCRIPTION
**Pull request by rui-mo**
This PR:

- refined docs on Velox backend.
- changed the name and version id of Gluten compiled jar.

https://github.com/oap-project/gluten/issues/13.

### CODE REVIEW

**Comment by rui-mo** [comment]
@marin-ma please take a review, thanks.

**Comment by rui-mo** [comment]
Verified on Velox backend. LGTM.

**Comment by rui-mo** [comment]
Verified on Velox backend. LGTM.

File: docs/ArrowInstallation.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Please make sure your cmake version is qualified based on the prerequisite.
 # Arrow
 ``` shell
 git clone https://github.com/oap-project/arrow.git
-cd arrow && git checkout arrow-4.0.0-oap
+cd arrow && git checkout arrow-7.0.0-oap
 mkdir -p arrow/cpp/release-build
```
Comment (QiangCai) [inline docs/ArrowInstallation.md:34]:
```suggestion
mkdir -p cpp/release-build
```

**Comment by zzcclp** [comment]
LGTM.

**Pull request merged**